backends-common/cassandra/src/test/java/org/apache/james/backends/cassandra/DockerCassandra.java

```diff
@@ -45,17 +49,21 @@
 
     @SuppressWarnings("resource")
     public DockerCassandra() {
+        this("cassandra_3_11_3", Function.identity());
+    }
+
+    public DockerCassandra(String imageName, Function<DockerfileBuilder, DockerfileBuilder> additionalSteps) {
```

Comment:
could you define a type instead of using raw Function?
I really think using Function should be avoided